src/test/scala/com/dwolla/lambda/cloudformation/CloudFormationCustomResourceHandlerSpec.scala

```diff
@@ -1,21 +1,23 @@
 package com.dwolla.lambda.cloudformation
 
-import java.io.{OutputStream, PrintWriter, StringWriter}
+import java.io._
 
+import cats.effect._
 import com.amazonaws.services.lambda.runtime.Context
 import com.amazonaws.util.StringInputStream
 import com.dwolla.lambda.cloudformation.SampleMessages._
 import com.dwolla.testutils.exceptions.NoStackTraceException
-import org.json4s.JsonAST.{JArray, JString}
-import org.json4s.ParserUtil.ParseException
-import org.mockito.Matchers
+import io.circe._
+import io.circe.syntax._
+import io.circe.generic.auto._
+import org.mockito._
 import org.slf4j.Logger
 import org.specs2.concurrent.ExecutionEnv
 import org.specs2.mock.Mockito
 import org.specs2.mutable.Specification
 import org.specs2.specification.Scope
 
-import scala.concurrent.{ExecutionContext, Future, Promise}
+import scala.concurrent._
 
 class CloudFormationCustomResourceHandlerSpec(implicit ee: ExecutionEnv) extends Specification with Mockito {
```

Comment:
Would it make sense to add a test that showed the usage of `CatsAbstractCustomResourceHandler`?

Reply:
Good idea; addressed in eafa3a5